### PR TITLE
Fix pad definition docs and types

### PR DESCRIPTION
**Pull request by bblaszkow06**

### CODE REVIEW

File: lib/membrane/element/base/mixin/sink_behaviour.ex [hunk 1]
```diff
@@ -27,13 +27,15 @@ defmodule Membrane.Element.Base.Mixin.SinkBehaviour do
             ) :: CommonBehaviour.callback_return_t()
 
   @doc """
-  Macro that defines known input pads for the element type.
+  Macro that defines input pads for the element type.
```
Comment (mat-hek) [inline lib/membrane/element/base/mixin/sink_behaviour.ex]: Remove that 'type'

File: lib/membrane/element/base/mixin/source_behaviour.ex [hunk 2]
```diff
@@ -43,13 +43,15 @@ defmodule Membrane.Element.Base.Mixin.SourceBehaviour do
             ) :: CommonBehaviour.callback_return_t()
 
   @doc """
-  Macro that defines known output pads for the element type.
+  Macro that defines output pads for the element type.
```
Comment (mat-hek) [inline lib/membrane/element/base/mixin/source_behaviour.ex]: Again, remove `type` please

File: lib/membrane/element/pad.ex [hunk 3]
```diff
@@ -24,7 +25,8 @@ defmodule Membrane.Element.Pad do
   @type name_t :: atom
 
   @typedoc """
-  Defines possible pad directions:
+  Defines possible pad directions.
```
Comment (mat-hek) [inline lib/membrane/element/pad.ex]: Please put the colon back!

File: lib/membrane/element/base/mixin/common_behaviour.ex [hunk 4]
```diff
@@ -23,7 +23,7 @@ defmodule Membrane.Element.Base.Mixin.CommonBehaviour do
   Generated by `Membrane.Element.Base.Mixin.SinkBehaviour.def_input_pads/1`
   and `Membrane.Element.Base.Mixin.SourceBehaviour.def_output_pads/1` macro.
   """
-  @callback membrane_pads() :: [{Pad.name_t(), Core.Element.PadsSpecsParser.parsed_pad_specs_t()}]
+  @callback membrane_pads() :: [{Pad.name_t(), Element.Pad.description_t()}]
```
Comment (mat-hek) [inline lib/membrane/element/base/mixin/common_behaviour.ex]: Afaik this returns a map

Comment (bblaszkow06) [reply lib/membrane/element/base/mixin/common_behaviour.ex]: You don't know far enough :stuck_out_tongue: 
```
iex(1)> Membrane.Element.FFmpeg.H264.Decoder.membrane_pads()
[
  output: %{
    availability: :always,
    caps: {Membrane.Caps.Video.Raw, [format: in([:I420, :I422]), aligned: true]},
    direction: :output,
    mode: :pull
  },
  input: %{
    availability: :always,
    caps: {Membrane.Caps.Video.H264,
     [stream_format: :byte_stream, alignment: :au]},
    demand_unit: :buffers,
    direction: :input,
    mode: :pull
  }
]
```

Comment (mat-hek) [reply lib/membrane/element/base/mixin/common_behaviour.ex]: :o

**Pull request merged**